draft-ietf-ppm-dap.md

```diff
@@ -751,7 +751,13 @@ To get the aggregate of a batch, the Collector issues a query specifying the
 batch ID of interest (see {{query}}). The Collector may not know which batch ID
 it is interested in; in this case, it can also issue a query of type
 `current_batch`, which allows the Leader to select a recent batch to aggregate.
-The Leader SHOULD select a batch which has not yet began collection.
+The Leader MUST select a batch which has not yet began collection.
+
+It is RECOMMENDED that Collectors which use `current_batch` queries generate &
+store collection job IDs as durably as they will store the eventual collection
```

Comment:
"as durably"?

Reply:
See the RECOMMEND vs MUST thread for more detail: what I'm trying to get at is that a collection job ID, even for an in-process collection job, is equally "important" as the eventual results from that collection job. Specifically, "important" in the sense that losing either would lose the results for the batch selected by the query, since in a `current-batch` query each batch is (now) assigned to exactly one collection job.

I use the term "durably" here in the sense of "durable storage", e.g. to suggest that the collection job ID be stored to the same datastore as the eventual collection result (as opposed to being stored in memory only, I suppose). Maybe there's a better way to phrase this?

Reply:
My chief complaint is that it's grammatically awkward. (When I get the tokens "as durably as", the next token I predict is "possible".)